### ISSUE TITLE
[Feature Request] Mode 0 and Mode 2 fonts

### ISSUE THREAD

**Issue by brknsoul** [damianvila/font-cpc464#2]
Any chance for Mode 0 and Mode 2 fonts?

Thanks for this!

**Comment by damianvila** [comment]
> Any chance for Mode 0 and Mode 2 fonts?
> 
> Thanks for this!

I'm honestly not too familiar with the CPC 464. Does it use a different font for the other modes? All the resources I saw just show one font. If you can provide a reference for the other fonts, I may be able to create those variations.

**Comment by brknsoul** [comment]
Mode 0
![image](https://github.com/damianvila/font-cpc464/assets/1969381/830d784f-2c39-487f-b7b2-7efe834327a9)

Mode 1
![image](https://github.com/damianvila/font-cpc464/assets/1969381/6df5e0e8-c8b4-4037-a4b6-d113516f1d00)

Mode 2 (i found one here https://retrofun.pl/2021/06/01/amstrad-cpc-international-font/ )
![image](https://github.com/damianvila/font-cpc464/assets/1969381/d91d4c60-aef8-4a33-9806-80dd169cc0df)

**Comment by damianvila** [comment]
Mmm... To me it seems that it's the same font, but with the characters stretched or shrunk. Since the pixel fonts are based in a "pixel" unit (a vector square), maybe modifying that character could get the desired result.
Let me see what I can do.

**Comment by brknsoul** [comment]
This might help a little
Mode 0 160×200
![image](https://github.com/damianvila/font-cpc464/assets/1969381/416c4850-6dea-4c49-8c5d-6d7384acee74)

Mode 2 640x200
![image](https://github.com/damianvila/font-cpc464/assets/1969381/83152b58-5c57-4183-b215-7d75bf5a3fd5)

Seems to be the same font as Mode 1, just doubled or halved horizontally.
![image](https://github.com/damianvila/font-cpc464/assets/1969381/b03fc09f-c287-4270-a577-0002effc3c6a)

And in case you're interested, I've used this font for an Amstrad Theme in the AI Image Generation software SD.Next;
![image](https://github.com/damianvila/font-cpc464/assets/1969381/e3cb43c6-827e-4936-93ca-cfb03176a330)

**Comment by damianvila** [comment]
> Seems to be the same font as Mode 1, just doubled or halved horizontally.

Yes, that seems to be the case. That may simplify things a bit. I only want to ask you to be a bit patient, as I'm with other projects at the moment, but as soon as I can review this, I'll try to make these versions. I'll leave this issue open, to remind me. Cheers!

**Comment by brknsoul** [comment]
Nah, it's all cool, just though it'd be handy for other looking for Amstrad fonts, just for completionist's sake.
